### PR TITLE
Master

### PR DESCRIPTION
**Pull request by dimante**
needed change to prevent \' showing on screen

### CODE REVIEW

File: e107_themes/landingzero/languages/English.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 define("LAN_LZ_THEME_11", "Phone");
 define("LAN_LZ_THEME_12", "Your message here...");
 define("LAN_LZ_THEME_13", "Get in Touch");
-define("LAN_LZ_THEME_14", "We love feedback. Fill out the form below and we\'ll get back to you as soon as possible.");
+define("LAN_LZ_THEME_14", "We love feedback. Fill out the form below and we&apos;ll get back to you as soon as possible.");
```
Comment (CaMer0n) [inline e107_themes/landingzero/languages/English.php]: Thanks. Did you try replacing `we\'ll ` with `we'll`

**Comment by dimante** [comment]
Yes that did not work on the display either the code in there with \' is
wrong though period...


Best Regards,

John Gates, CISSP

*Let’s Connect!*

<https://twitter.com/johngatesIII>   <http://www.linkedin.com/in/JohnGates>

File: e107_themes/landingzero/languages/English.php (hunk 1 above)
Comment (dimante) [reply e107_themes/landingzero/languages/English.php]: I did and it did not display properly... What I can tell you is the '\' shows in the rendered text and is wrong ;-)

Comment (dimante) [reply e107_themes/landingzero/languages/English.php]: that was supposed to be the backslash char in the ''

**Comment by Jimmi08** [comment]
@yesszus Could you please look at this?  You language could have the same problem.  There shouldn't be `&apos;`   correct fix should be `we'll`
Thanks

**Comment by yesszus** [comment]
@Jimmi08 tested and work this `&apos;` and `we'll` without display error.

**Pull request closed without merging**